yandex/geocoder.go

```diff
@@ -0,0 +1,159 @@
+// Package yandex is a geo-golang based Yandex Maps Location API
+// https://tech.yandex.com/maps/doc/geocoder/desc/concepts/About-docpage/
```

Comment:
Can you please remove this line, and add below in README after line 27?
+ [Yandex](https://tech.yandex.com/maps/doc/geocoder/desc/concepts/About-docpage/)

Reply:
@codingsince1985 done